numba/cuda/dispatcher.py

```diff
@@ -96,6 +96,26 @@ def __init__(self, py_func, argtypes, link=None, debug=False,
         if self.cooperative:
             link.append(get_cudalib('cudadevrt', static=True))
 
+        cuda_fp16_math_funcs = ['hsin_wrapper', 'hcos_wrapper',
```

Comment:
I'd suggest moving this to global scope - in future the list will get longer. It would also be nice to avoid baking the `_wrapper` part into the striongs so we can just list the actual function names.